docker/bin/push2deis.sh

```diff
@@ -8,7 +8,22 @@
 
 set -ex
 
-deis pull "$DOCKER_REPOSITORY:$GIT_COMMIT" -a $DEIS_APPLICATION
+BIN_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source $BIN_DIR/set_git_env_vars.sh
+
+DEIS_BIN="${DEIS_BIN:-deis}"
+
+if [[ "$DEIS_BIN" == "deis2" ]]; then
+    if [[ "$DEMO_MODE" == "true" ]]; then
+        DOCKER_REPO="mozorg/bedrock_demo:${GIT_COMMIT}"
+    else
+        DOCKER_REPO="mozorg/bedrock:${BRANCH_NAME}-${GIT_COMMIT}"
+    fi
+else
+    DOCKER_REPO="$DOCKER_PRIVATE_REPO:${GIT_COMMIT}"
```

Comment:
@jgmize should correct me if I'm wrong, but last I talked to him about it we were not planning to have private registries for deis2 as the efficiency gains for deis1 weren't that great.